DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -205,10 +204,9 @@ private async Task FetchConfigAsyncWithTask()
 
                 logger.LogError(finalError.ErrorResponse.Message);
                 dvcEventArgs.Errors.Add(finalError);
-            } catch (System.ArgumentNullException e) {
-                // This error can be thrown if the config is in an invalid format 
-                // from the method SetConfig
-                logger.LogError(e.Message);
+            } catch (System.Exception e) {
```

Comment:
is there not a specific error type you can catch here?

Reply:
I don't think so. The exception is thrown from https://github.com/DevCycleHQ/assemblyscript-json/blob/main/assembly/decoder.ts#L144. And I think cause it's from assemblyscript, it ends up begin a generic `System.Exception`

Reply:
catching `WasmtimeException` now instead.